core/state/snapshot/snapshot.go

```diff
@@ -604,13 +579,28 @@ func diffToDisk(bottom *diffLayer) *diskLayer {
 			if len(data) > 0 {
 				rawdb.WriteStorageSnapshot(batch, accountHash, storageHash, data)
 				base.cache.Set(append(accountHash[:], storageHash[:]...), data)
+
 				snapshotCleanStorageWriteMeter.Mark(int64(len(data)))
+				if metrics.EnabledExpensive {
+					if oldbytes := len(rawdb.ReadStorageSnapshot(base.diskdb, accountHash, storageHash)); oldbytes == 0 {
+						snapshotStorageCountGrowthMeter.Mark(1)
+						snapshotStorageBytesGrowthMeter.Mark(2*common.HashLength + int64(len(data)))
+					} else {
+						snapshotStorageBytesGrowthMeter.Mark(int64(len(data) - oldbytes))
+					}
+					snapshotFlushStorageItemMeter.Mark(1)
```

Comment:
`snapshotFlushStorageItemMeter` and `snapshotFlushStorageSizeMeter` should be tracked even `EnabledExpensive` is not enabled, right?

Reply:
Ugh, good catch